t/31-client.t

```diff
@@ -105,4 +105,19 @@ $options{json} = 1;
 stderr_like \&wrong_call, qr/$code.*Not Found/, 'Error reported for undocumented "json" parameter';
 is $ret, undef, 'undef shows error for undocumented parameter';
 
+subtest 'YAML Templates' => sub {
+    my $template_openqa = "$FindBin::Bin/data/job-templates/openqa.yaml";
+    my @common_args     = qw(job_templates_scheduling/1 post schema=JobTemplates-01.yaml preview=1);
+    my $ret;
+    stderr_is { $ret = run(\%options, qw(job_groups post name=Test)) } '', 'create job group';
+    warn "RET is [$ret]";
+    is $ret, { id => 1 }, 'create job group';
```

Comment:
I'd remove that unconditional warning. If `$res` is not the expected value the actual value should be logged by the subsequent line if you use `is_deeply`.

Reply:
Oh, I already removed it. This was added just to debug the tests, as they're working in my local system.